driver/src/main.rs

```diff
@@ -63,9 +63,15 @@ struct Options {
     #[structopt(long, env = "SOLVER_TYPE", default_value = "NaiveSolver")]
     solver_type: SolverType,
 
+    /// JSON encoded object of back-up price information in case the oracle functionality is
+    /// not working.
     #[structopt(long, env = "PRICE_FEED_INFORMATION", default_value = "{}")]
     backup_token_data: TokenData,
 
+    /// Number of seconds the solver should maximally use for the optimization process
+    #[structopt(long, env = "SOLVER_TIME_LIMIT", default_value = "150")]
+    solver_time_limit: u32,
```

Comment:
Since this and TokenData only apply to the Standard and Fallback Solver can we make it a field on the enum subtypes instead of passing more and more data into create_price_finder?